langchain4j-azure-cosmos-no-sql/src/main/java/dev/langchain4j/store/embedding/azure/cosmos/no/sql/AzureCosmosDBNoSqlDocument.java

```diff
@@ -0,0 +1,20 @@
+package dev.langchain4j.store.embedding.azure.cosmos.no.sql;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class AzureCosmosDBNoSqlDocument {
```

Comment:
nit: is there a reason this class should be public?

Reply:
Fixed in new iteration.